index.html

```diff
@@ -8,7 +8,87 @@
 </head>
 
 <body>
-   
+    <nav>
+        <img src="Logo.svg" alt="">
+        <p>Product</p>
+        <p>Customers</p>
+        <p>Pricing</p>
+        <p>Resources</p>
+        <button><img src="Button1.svg" alt=""></button>
+        <button><img src="Button2.svg" alt=""></button>
+    </nav>
```

Comment:
тег `nav` суто для навігації, тобто в середині нього мають бути посилання але можна додавати й інші елементи, якщо вони пов'язані з цією навігацією
```suggestion
    <div>
        <a href="/">
            <img src="Logo.svg" alt="">
        </a>
        <nav>
            <a href="#product">Product</a>
            <a href="#product">Customers</a>
            <a href="#product">Pricing</a>
            <a href="#product">Resources</a>
        </nav>
        <button><img src="Button1.svg" alt=""></button>
        <button><img src="Button2.svg" alt=""></button>
    </div>
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav
https://www.w3schools.com/tags/tag_nav.asp